src/services/tx/txEvents.ts

```diff
@@ -12,26 +10,29 @@ export enum TxEvent {
   SIGNATURE_PROPOSE_FAILED = 'SIGNATURE_PROPOSE_FAILED',
   EXECUTING = 'EXECUTING',
   MINING = 'MINING',
+  MINING_MODULE = 'MINING_MODULE',
   MINED = 'MINED',
   REVERTED = 'REVERTED',
   FAILED = 'FAILED',
   SUCCESS = 'SUCCESS',
 }
 
+type Id = { txId: string; groupKey?: string } | { txId?: string; groupKey: string }
+
 interface TxEvents {
-  [TxEvent.CREATED]: { tx: SafeTransaction }
-  [TxEvent.SIGNED]: { txId?: string; tx: SafeTransaction }
-  [TxEvent.SIGN_FAILED]: { txId?: string; tx: SafeTransaction; error: Error }
-  [TxEvent.PROPOSE_FAILED]: { tx: SafeTransaction; error: Error }
-  [TxEvent.PROPOSED]: { txId: string; tx: SafeTransaction }
-  [TxEvent.SIGNATURE_PROPOSE_FAILED]: { txId: string; tx: SafeTransaction; error: Error }
-  [TxEvent.SIGNATURE_PROPOSED]: { txId: string; tx: SafeTransaction }
-  [TxEvent.EXECUTING]: { txId: string; tx: SafeTransaction; batchId?: string }
-  [TxEvent.MINING]: { txId: string; txHash: string; tx: SafeTransaction; batchId?: string }
-  [TxEvent.MINED]: { txId: string; receipt: ContractReceipt; tx: SafeTransaction; batchId?: string }
-  [TxEvent.REVERTED]: { txId: string; error: Error; receipt: ContractReceipt; tx?: SafeTransaction; batchId?: string }
-  [TxEvent.FAILED]: { txId: string; error: Error; tx?: SafeTransaction; batchId?: string }
-  [TxEvent.SUCCESS]: { txId: string; batchId?: string }
+  [TxEvent.SIGNED]: { txId?: string }
+  [TxEvent.SIGN_FAILED]: { txId?: string; error: Error }
+  [TxEvent.PROPOSE_FAILED]: { error: Error }
+  [TxEvent.PROPOSED]: { txId: string }
+  [TxEvent.SIGNATURE_PROPOSE_FAILED]: { txId: string; error: Error }
+  [TxEvent.SIGNATURE_PROPOSED]: { txId: string }
+  [TxEvent.EXECUTING]: Id
+  [TxEvent.MINING]: Id & { txHash: string }
+  [TxEvent.MINING_MODULE]: Id & { txHash: string; message: string }
```

Comment:
We agreed to remove the message right?

Reply:
Ah, good catch! Forgot to remove it after resetting the commit